### PR TITLE
Update for working romstage

### PR DESCRIPTION
**Pull request by IgorBagnucki**

### CODE REVIEW

**Comment by crgeddes** [comment]
It's very cool to see your team's progress!

File: devnotes/documentation.md [hunk 1]
```diff
@@ -150,39 +149,33 @@ In order to build coreboot image, follow the steps below:
...
-   ```
-   git clone git@github.com:qemu/qemu.git
-   # or HTTPS alternatively
-   git clone https://github.com/qemu/qemu.git
-   ```
-2. Build the QEMU ppc64 version
-   ```
-   cd qemu
-    ./configure --target-list=ppc64-softmmu && make
+## Memory configurations
```
Comment (macpijan) [inline devnotes/documentation.md:166]: Do we explain somewhere the CPU count and the exact CPU we use?

File: devnotes/documentation.md [hunk 2]
```diff
@@ -150,39 +149,33 @@ In order to build coreboot image, follow the steps below:
...
+Following RAM configurations were tested and are proved to be properly initialized.
+   ```
+   MCS0, MCA0
+      DIMM0: 1Rx4 16GB
```
Comment (macpijan) [inline devnotes/documentation.md:170]: Do we have some more details on the exact parts that were used?

Comment (IgorBagnucki) [reply devnotes/documentation.md]: I have added information from the labels, but I don't know the details about about `1Rx8 8GB`

File: devnotes/documentation.md (hunk 1 above)
Comment (IgorBagnucki) [reply devnotes/documentation.md]: I have added a short note, what CPU are we using. Is it enough? I couldn't find details, what CPU is used in any of our documents.

File: devnotes/documentation.md (hunk 2 above)
Comment (macpijan) [reply devnotes/documentation.md]: I beleive the most important fact is that it is a **single** CPU setup and dual CPU setup is not supported currently

Comment (IgorBagnucki) [reply devnotes/documentation.md]: I have added this information.

**Pull request merged**